### PR TITLE
MicaForEveryone.MicaForEveryone version 1.0.6.3

### PR DESCRIPTION
**Pull request by xmine64**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/64454)

### CODE REVIEW

**Comment by xmine64** [comment]
I updated installer so now it supports /SILENT argument, Please rerun the test.

File: manifests/m/MicaForEveryone/MicaForEveryone/1.0.6.3/MicaForEveryone.MicaForEveryone.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MicaForEveryone.MicaForEveryone
+PackageVersion: 1.0.6.3
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/MicaForEveryone/MicaForEveryone/releases/download/v1.0.6.3/MicaForEveryone-v1.0.6.3-x64-Installer.exe
+  InstallerSha256: 40E57A9843EB8ED54A6C6E371B792743084F4642D12FD1852B4CE200D697F9E1
```
Comment (ItzLevvie) [inline manifests/m/MicaForEveryone/MicaForEveryone/1.0.6.3/MicaForEveryone.MicaForEveryone.installer.yaml:10]:
```suggestion
  InstallerSha256: 047D67FB0075E693CA2A4101F4204BC0C503BCBC3FD1DC7072D08FBC25264641
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by xmine64** [comment]
> I am sorry to report that the Sha256 Hash does not match the installer.

Yeah I'm sorry, my installer wasn't working silent and Installer Validation on Azure failed so I updated installer with `/SILENT` support but I forgot to update hash. Thanks suggested edit seems ok.

image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**